scripts/tests/create_project_table_test.py

```diff
@@ -0,0 +1,139 @@
+"""
+Unit tests create_project_table.py.
+
+This suite tests functions for extracting metadata, generating markdown table rows, 
+processing multiple README files, and inserting rows into tables, using mocks to simulate file operations.
+"""
+
+import unittest
+from unittest.mock import MagicMock, patch
+from pathlib import Path
+import sys
```

Comment:
Maintain alphabetical order in imports

```suggestion
from pathlib import Path
import sys
import unittest
from unittest.mock import MagicMock, patch
```